src/helpers/figures/charts/gauge_chart_rendering.ts

```diff
@@ -228,7 +228,7 @@ export function getGaugeRenderingConfig(
     ({ width: titleWidth, height: titleHeight } = computeTextDimension(
       ctx,
       runtime.title.text,
-      { ...runtime.title, fontSize: GAUGE_TITLE_FONT_SIZE },
+      { ...runtime.title, fontSize: runtime.title.fontSize ?? GAUGE_TITLE_FONT_SIZE },
```

Comment:
nitpick: maybe rename GAUGE_TITLE_FONT_SIZE to DEFAULT_GAUGE_TITLE_FONT_SIZE now